Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst

```diff
@@ -0,0 +1,80 @@
+.. include:: /Includes.rst.txt
+.. index:: Localization; TypoScript
+
+==========
+TypoScript
+==========
+
+Output localized strings with Typoscript
+========================================
+
+The :ref:`getText property LLL <t3tsref:data-type-gettext-lll>` can be
+used to fetch translations from a translation file and output it
+in the current language:
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+   lib.blogListTitle = TEXT
+   lib.blogListTitle.data = LLL : EXT:blog_example/Resources/Private/Language/locallang.xlf:blog.list
+
+TypoScript conditions based on the current language
+===================================================
+
+The condition function
+:ref:`siteLanguage <t3tsref:condition-functions-in-frontend-context-function-siteLanguage>`
+can be used to provide certain TypoScript configurations only for certain
+languages. You can query for any property of the language in the
+site configuration.
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+   [siteLanguage("locale") == "de_CH"]
+       lib.something.value = This site has the locale "de_CH"
+   [end]
+
+   [siteLanguage("title") == "Italy"]
+       lib.languageTitle.value = This site has the title "Italy"
```

Comment:
Please adjust to the same lib. Also lib.something = TEXT should be added for clarity (outside the condition with a default value).